src/openrct2-ui/windows/Staff.cpp

```diff
@@ -967,8 +967,7 @@ void window_staff_options_tab_paint(rct_window* w, rct_drawpixelinfo* dpi)
         return;
 
     rct_widget* widget = &w->widgets[WIDX_TAB_2];
-    int32_t x = widget->left + w->windowPos.x;
-    int32_t y = widget->top + w->windowPos.y;
+    auto screenCoords = w->windowPos + ScreenCoordsXY{ widget->left, widget->top };
```

Comment:
Can you move this to right before the `gfx_draw_sprite` call?
It's not used anywhere, so we keep it close to its usage